modules/custom/apigee_kickstart_enhancement/apigee_kickstart_enhancement.module

```diff
@@ -282,7 +282,7 @@ function _apigee_kickstart_enhancement_check_app_warnings(AppInterface $app, Ent
   foreach ($app->getCredentials() as $credential) {
     if ($credential->getStatus() === AppCredential::STATUS_REVOKED) {
       $args = [
-        '@app' => $entity_type->getLowercaseLabel(),
+        '@app' => mb_strtolower($entity_type->getSingularLabel()),
```

Comment:
Do we still need the `md_strtolower` here?

Reply:
Thanks @arshad , we do needed because the site administrator can override the name of the developer and team app entities, so it's user input.